scripts/dig.js

```diff
@@ -0,0 +1,27 @@
+var b = pizza.open('https://toolbox.googleapps.com/apps/dig/');
+
+pizza.sleep(1000);
+// b.waitPageLoad();  // script has been getting hung up
+
+var domains = ['walmart.com', 'amazon.com'];
+
+b.type('#domain', domains[Math.floor((Math.random() * domains.length - 1) + 1)]);
+
+var records = ['A', 'AAAA', 'ANY', 'CNAME', 'MX', 'NS', 'PTR', 'SOA', 'SRV', 'TXT'];
+b.click('xpath://input[@value="' + records[5] + '"]');
+
+b.waitForVisible('#response-text');
+
+var html = b.getInnerHTML('xpath://*[@id="response-text"]');
```

Comment:
The css selector version is probably easier to read:

var html = b.getInnerHTML('#response-text');